DeploymentExperiences.md

```diff
@@ -0,0 +1,7 @@
+# Deployment
+
+1. Repo Link: [Intro-Django](https://github.com/wvandolah/Intro-Django)
+2. Deployment Link: [Heroku](https://fast-waters-75018.herokuapp.com/)  
+
+## Deployment Experience
+Deployment to heroku went super smooth.  Read thru all the [getting-started-with-python](https://devcenter.heroku.com/articles/getting-started-with-python) first then followed the [README](./README.md) provided in the repo.  I did get an error "No directory at: {}" after following the steps to setup the whitenoise genorater.  I fixed this by adding an empty staticfiles folder. I need to do more research on what exactly this middleware is  doing.  Overall, the actually deployment to heroku is very straight forward and simple.  
```

Comment:
Find out what the 'no directory error' is all about. But great job other than that! Solid work William!